packages/nextra/package.json

```diff
@@ -88,10 +89,16 @@
   "peerDependencies": {
     "react": ">=16.13.1"
   },
+  "prettier": {
```

Comment:
It’s better to have 1 top level prettier config, rather than add prettier config for each monorepo package

Reply:
It is used for [prettier-plugin-tailwindcss](https://github.com/tailwindlabs/prettier-plugin-tailwindcss)